pytest_pootle/fixtures/views.py

```diff
@@ -134,21 +134,18 @@ def project_views(request, client):
     return test_type, project, response.wsgi_request, response, kwargs
 
 
-@pytest.fixture(params=TP_VIEW_TESTS.keys())
-def tp_view_test_names(request):
-    return request.param
-
-
 @pytest.fixture(params=(None, 'member', 'admin'))
-def tp_view_usernames(request):
+def username(request):
```

Comment:
can we make this plural - so its clear its params - also ``site_users`` or similar might be useful - im thinking of adding the password into the fixture